docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ghcr.io/gazebosim/gz-jetty:main
+ENV LANG=C
+ENV LC_ALL=C
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+    && apt-get install -y curl
+ENV ROS_APT_SOURCE_VERSION=1.1.0
+RUN curl -L -o /tmp/ros2-apt-source.deb "https://github.com/ros-infrastructure/ros-apt-source/releases/download/${ROS_APT_SOURCE_VERSION}/ros2-apt-source_${ROS_APT_SOURCE_VERSION}.$(. /etc/os-release && echo ${UBUNTU_CODENAME:-${VERSION_CODENAME}})_all.deb"
+RUN dpkg -i /tmp/ros2-apt-source.deb
```

Comment:
Can you add the `testing` repos?
https://docs.ros.org/en/rolling/Installation/Testing.html#deb-testing-repository